yesod-core/yesod-core.cabal

```
@@ -26,7 +26,8 @@ library
     hs-source-dirs: src
 
     build-depends:   base                  >= 4.10     && < 5
-                   , aeson                 >= 1.0
+                   , aeson                 >= 2.2
+                   , attoparsec-aeson      >= 2.2
```

```suggestion
                   , aeson                 >= 1.0
                   , attoparsec-aeson      >= 2.1
```

This ought to fix things